accounts/models.py

```diff
@@ -73,6 +79,47 @@ class Meta:
         ordering = ["name"]
 
 
+class GoogleAuthenticatorTOTP(models.Model):
+    """OTP Secret model for time-based OTP.
+
+    Currently, we only support Google authenticator format. Please see:
+    https://github.com/google/google-authenticator/wiki/Key-Uri-Format
+    """
+
+    issuer = "Lookit"
```

Comment:
Is this the label that shows up in the Authenticator app? Do we want it to say e.g. Lookit vs. Lookit-staging vs. Lookit-dev?

Reply:
Excellent point!